runtime/tests/shared/AttachedDataTest.cpp

```diff
@@ -1427,17 +1426,16 @@ AttachedDataTest::CorruptAttachedData(J9JavaVM *vm)
 		rc = FAIL;
 		goto _exit;
 	}
-	strncpy((char *)data.address, content, strlen(content));
-	data.address[strlen(content)] = '\0';
 	data.length = strlen(content) + 1;
+	memcpy((char *)data.address, content, data.length); /* copy NUL */
```

Comment:
No, `data` was cleared above, but not the block referenced by `data.address`.

Reply:
Then the commit comment is wrong, since the destination isn't in a block that has already been cleared. Plus my point was that since the code is explicitly copying the NULL, it doesn't matter if the destination is cleared or not.

Reply:
Corrected commit message.